GitExercise_1/App.config

```diff
@@ -1,6 +1,14 @@
-﻿<?xml version="1.0" encoding="utf-8" ?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
     <startup> 
         <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7.2" />
     </startup>
+  <runtime>
+    <assemblyBinding xmlns="urn:schemas-microsoft-com:asm.v1">
```

Comment:
I would suggest to avoid using .net framework. .net Core is more preferred

Reply:
I used console application,  I just didnt know, console application .net Core exists ). I'll use .net Core app futher.